moped-editor/src/components/ActivityMetrics.js

```diff
@@ -21,6 +23,6 @@ export default function ActivityMetrics({ children }) {
         error
       );
     });
-  }, [setLastSeen, user]);
+  }, [setLastSeen, isLoggedIn]);
```

Comment:
thanks @mddilley for flagging that the `user` object is not a stable reference. i have updated the hook to test for the existence of the user object, which reliably works.